.github/ISSUE_TEMPLATE/testplan.md

```diff
@@ -382,6 +382,45 @@ and Linux `tsh` binaries are tested for FIDO2 items.
   - [ ] `tsh touchid ls` works
   - [ ] `tsh touchid rm` works (careful, may lock you out!)
 
+### Hardware Key Login
+
+Hardware Key Login is an Enterprise feature and is not available for OSS.
+
+On Linux, hardware key support requires that `tsh` and `teleport` are both built with libpcsclite. Ask for a statically-built `tsh` and `teleport` binary for realistic tests.
```

Comment:
What does statically-built mean?

Reply:
It just means to use a release build (`-alpha` in this case), or create one with `make -C build.assets` rather than the usual `make` from `/teleport`. These builds use `libpcsclite` as a statically-linked library rather than a dynamically-linked library (C stuff).

I'm copying the phrasing used above for libfido which has a similar build process. @codingllama can you confirm that this is what you meant / I'm using this phrasing correctly?

Reply:
Gotcha, I just hadn't heard that precise term before and wondered if it was different from "static build"/"statically linked build". 

Now that you mention it here, though, maybe you want to include a note on how to create the static build rather than saying to ask for it.

Reply:
my2c. Currently, we include `libpcsc` in all Linux builds. I'd just say that this feature should be tested with our "release build" from Drone rather than dev build (`git clone teleport && make`)

Reply:
Saying "release build" sounds fine to me. Feel free to tweak the libfido2 sentence if you guys find it difficult to read.